rt/external/apache-http/src/org/apache/http/message/BasicLineFormatter.java

```diff
@@ -71,7 +71,7 @@
      * The instance here provides non-customized, default behavior.
      */
     public final static BasicLineFormatter DEFAULT = new BasicLineFormatter();
-
+    public final static BasicLineFormatter INSTANCE = new BasicLineFormatter();
```

Comment:
Can you add a "// RoboVM: added, see issue #834 " above this line? I'll merge it.